src/app.ts

```
@@ -41,5 +124,77 @@ app.on(["GET", "POST"], "/api/uploadthing", isAuth);
+          console.error(`Redis subscription error for ${connectionId}:`, err);
+
+          // Try to resubscribe after delay
+          setTimeout(subscribe, 5000);
+        }
+      };
+
+      subscribe();
+
+      redisSubscriber.on("message", onMessage);
+      let keepAliveDelay = 15000; // Start with 15s
+      let consecutiveErrors = 0;
```

ESLint

[error] 'consecutiveErrors' is assigned a value but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)